### ISSUE TITLE
while setting timezone, no /etc directory

### ISSUE THREAD

**Issue by dufresnep** [Vanilla-OS/vanilla-installer#195]
I have to Log window opened, but cannot copy from it!
/var/log/installer is empty

/mnt/a/etc/timezone : no such file

vanillaos@vanillaos:~$ dir /mnt/a/
```
System.map-5.19.0-31-generic  lost+found
config-5.19.0-31-generic      memtest86+.bin
efi			      memtest86+.elf
grub			      memtest86+_multiboot.bin
initrd.img		      vmlinuz
initrd.img-5.19.0-31-generic  vmlinuz-5.19.0-31-generic
initrd.img.old		      vmlinuz.old
```

Somehow I suspect this is because I have both set a boot partition and a efi partition.

root@vanillaos:/var/log# cat syslog | grep -i timezone
```
Jun  4 21:57:53 vanillaos org.vanillaos.Installer.desktop[2078]: INFO:Installer::Processor:processing the following final data: [{}, {'language': 'fr_FR.UTF-8'}, {'keyboard': 'ca'}, {'timezone': {'region': 'America', 'zone': 'Toronto'}}, {'disk': {'disk': '/dev/sda', '/dev/sda1': {'fs': 'ext4', 'mp': '/boot', 'pretty_size': '1.95 GB', 'size': 2097152000}, '/dev/sda2': {'fs': 'fat32', 'mp': '/boot/efi', 'pretty_size': '1.95 GB', 'size': 2097152000}, '/dev/sda3': {'fs': 'btrfs', 'mp': '/', 'pretty_size': '43.45 GB', 'size': 46651146240}, '/dev/sda7': {'fs': 'btrfs', 'mp': '/', 'pretty_size': '43.45 GB', 'size': 46651146240}, '/dev/sda9': {'fs': 'btrfs', 'mp': '/home', 'pretty_size': '43.95 GB', 'size': 47185920000}, '/dev/sda8': {'fs': 'swap', 'mp': 'swap', 'pretty_size': '17.58 GB', 'size': 18874368000}}}, {'users': {'fullname': 'Paul Dufresne...
```

**Comment by dufresnep** [comment]
lsblk output:
```
root@vanillaos:/var/log# lsblk
NAME       MAJ:MIN RM   SIZE RO TYPE MOUNTPOINTS
loop0        7:0    0   1.5G  1 loop /rofs
loop1        7:1    0  53.3M  1 loop /snap/snapd/19361
loop2        7:2    0  63.5M  1 loop /snap/core20/1891
loop3        7:3    0     4K  1 loop /snap/bare/5
loop4        7:4    0  91.7M  1 loop /snap/gtk-common-themes/1535
loop5        7:5    0 349.7M  1 loop /snap/gnome-3-38-2004/140
loop6        7:6    0 242.9M  1 loop /snap/firefox/2710
sda          8:0    0 596.2G  0 disk 
├─sda1       8:1    0  43.4G  0 part 
├─sda2       8:2    0  43.4G  0 part 
├─sda3       8:3    0     2G  0 part /mnt/b
├─sda4       8:4    0   500M  0 part 
├─sda5       8:5    0   120G  0 part 
├─sda6       8:6    0     9G  0 part 
├─sda7       8:7    0     2G  0 part /mnt/a
├─sda8       8:8    0  17.6G  0 part 
└─sda9       8:9    0  43.9G  0 part 
sdb          8:16   1  28.8G  0 disk 
├─sdb1       8:17   1  28.8G  0 part 
│ └─ventoy 253:0    0   1.8G  1 dm   /cdrom
└─sdb2       8:18   1    32M  0 part 
sr0         11:0    1 700.4M  0 rom  /media/vanillaos/Mon disque
zram0      252:0    0   7.7G  0 disk [SWAP]
root@vanillaos:/var/log# 
```

mount:
```
root@vanillaos:/var/log# mount
sysfs on /sys type sysfs (rw,nosuid,nodev,noexec,relatime)
proc on /proc type proc (rw,nosuid,nodev,noexec,relatime)
udev on /dev type devtmpfs (rw,nosuid,relatime,size=8061648k,nr_inodes=2015412,mode=755,inode64)
devpts on /dev/pts type devpts (rw,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=000)
tmpfs on /run type tmpfs (rw,nosuid,nodev,noexec,relatime,size=1625092k,mode=755,inode64)
/dev/mapper/ventoy on /cdrom type iso9660 (ro,noatime,nojoliet,check=s,map=n,blocksize=2048,iocharset=utf8)
/dev/loop0 on /rofs type squashfs (ro,noatime,errors=continue)
/cow on / type overlay (rw,relatime,lowerdir=/filesystem.squashfs,upperdir=/cow/upper,workdir=/cow/work,xino=off)
securityfs on /sys/kernel/security type securityfs (rw,nosuid,nodev,noexec,relatime)
tmpfs on /dev/shm type tmpfs (rw,nosuid,nodev,inode64)
tmpfs on /run/lock type tmpfs (rw,nosuid,nodev,noexec,relatime,size=5120k,inode64)
cgroup2 on /sys/fs/cgroup type cgroup2 (rw,nosuid,nodev,noexec,relatime,nsdelegate,memory_recursiveprot)
pstore on /sys/fs/pstore type pstore (rw,nosuid,nodev,noexec,relatime)
efivarfs on /sys/firmware/efi/efivars type efivarfs (rw,nosuid,nodev,noexec,relatime)
bpf on /sys/fs/bpf type bpf (rw,nosuid,nodev,noexec,relatime,mode=700)
systemd-1 on /proc/sys/fs/binfmt_misc type autofs (rw,relatime,fd=29,pgrp=1,timeout=0,minproto=5,maxproto=5,direct,pipe_ino=18760)
debugfs on /sys/kernel/debug type debugfs (rw,nosuid,nodev,noexec,relatime)
hugetlbfs on /dev/hugepages type hugetlbfs (rw,relatime,pagesize=2M)
mqueue on /dev/mqueue type mqueue (rw,nosuid,nodev,noexec,relatime)
tracefs on /sys/kernel/tracing type tracefs (rw,nosuid,nodev,noexec,relatime)
/cow on /.system/usr type overlay (ro,relatime,lowerdir=/filesystem.squashfs,upperdir=/cow/upper,workdir=/cow/work,xino=off)
fusectl on /sys/fs/fuse/connections type fusectl (rw,nosuid,nodev,noexec,relatime)
configfs on /sys/kernel/config type configfs (rw,nosuid,nodev,noexec,relatime)
ramfs on /run/credentials/systemd-sysusers.service type ramfs (ro,nosuid,nodev,noexec,relatime,mode=700)
tmpfs on /tmp type tmpfs (rw,nosuid,nodev,relatime,inode64)
sunrpc on /run/rpc_pipefs type rpc_pipefs (rw,relatime)
tmpfs on /run/user/999 type tmpfs (rw,nosuid,nodev,relatime,size=1625088k,nr_inodes=406272,mode=700,uid=999,gid=999,inode64)
gvfsd-fuse on /run/user/999/gvfs type fuse.gvfsd-fuse (rw,nosuid,nodev,relatime,user_id=999,group_id=999)
/dev/sr0 on /media/vanillaos/Mon disque type iso9660 (ro,nosuid,nodev,relatime,norock,check=r,map=n,blocksize=2048,uid=999,gid=999,dmode=500,fmode=400,iocharset=utf8,uhelper=udisks2)
portal on /run/user/999/doc type fuse.portal (rw,nosuid,nodev,relatime,user_id=999,group_id=999)
/dev/sda7 on /mnt/a type ext4 (rw,relatime)
/dev/sda3 on /mnt/b type vfat (rw,relatime,fmask=0022,dmask=0022,codepage=437,iocharset=iso8859-1,shortname=mixed,errors=remount-ro)
binfmt_misc on /proc/sys/fs/binfmt_misc type binfmt_misc (rw,nosuid,nodev,noexec,relatime)
/var/lib/snapd/snaps/snapd_19361.snap on /snap/snapd/19361 type squashfs (ro,nodev,relatime,errors=continue,x-gdu.hide,x-gvfs-hide)
/var/lib/snapd/snaps/core20_1891.snap on /snap/core20/1891 type squashfs (ro,nodev,relatime,errors=continue,x-gdu.hide,x-gvfs-hide)
/var/lib/snapd/snaps/bare_5.snap on /snap/bare/5 type squashfs (ro,nodev,relatime,errors=continue,x-gdu.hide,x-gvfs-hide)
/var/lib/snapd/snaps/gtk-common-themes_1535.snap on /snap/gtk-common-themes/1535 type squashfs (ro,nodev,relatime,errors=continue,x-gdu.hide,x-gvfs-hide)
/var/lib/snapd/snaps/gnome-3-38-2004_140.snap on /snap/gnome-3-38-2004/140 type squashfs (ro,nodev,relatime,errors=continue,x-gdu.hide,x-gvfs-hide)
/var/lib/snapd/snaps/firefox_2710.snap on /snap/firefox/2710 type squashfs (ro,nodev,relatime,errors=continue,x-gdu.hide,x-gvfs-hide)
tmpfs on /run/snapd/ns type tmpfs (rw,nosuid,nodev,noexec,relatime,size=1625092k,mode=755,inode64)
nsfs on /run/snapd/ns/firefox.mnt type nsfs (rw)
/cow on /var/snap/firefox/common/host-hunspell type overlay (ro,noexec,noatime,lowerdir=/filesystem.squashfs,upperdir=/cow/upper,workdir=/cow/work,xino=off)
```

**Comment by dufresnep** [comment]
Although the installer did not finished, I have tried to reboot.

After reboot, I was pushed in grub command line.

So I did go to firmware settings, change boot order to boot Fedora on same disks, and in fedora "grub2-mkconfig -o grub.cfg" in the appropriate directory. This detected "Vanilla OS 22.10", and so I was able to boot it. First installation asked me to choose my packages, and tried to install them. I think it had almost finished when:
```
Locking apt
Locking apt-get
Locking dpkg
    ERROR: exec failed could not detect future root partition
```

I now have:
```
paul@vanilla:~$ lsblk
NAME                MAJ:MIN RM   SIZE RO TYPE  MOUNTPOINTS
loop0                 7:0    0  53,3M  1 loop  /snap/snapd/19361
loop1                 7:1    0  63,5M  1 loop  /snap/core20/1891
loop2                 7:2    0     4K  1 loop  /snap/bare/5
sda                   8:0    0 596,2G  0 disk  
├─sda1                8:1    0  43,4G  0 part  
├─sda2                8:2    0  43,4G  0 part  /.system/usr
│                                              /
├─sda3                8:3    0     2G  0 part  /boot/efi
├─sda4                8:4    0   500M  0 part  
├─sda5                8:5    0   120G  0 part  
├─sda6                8:6    0     9G  0 part  
│ └─cryptswap       253:0    0     9G  0 crypt [SWAP]
├─sda7                8:7    0     2G  0 part  /boot
├─sda8                8:8    0  17,6G  0 part  
│ └─cryptswap_rtmpq 253:1    0  17,6G  0 crypt [SWAP]
└─sda9                8:9    0  43,9G  0 part  /home
sdb                   8:16   1  28,8G  0 disk  
├─sdb1                8:17   1  28,8G  0 part  /media/paul/Ventoy
└─sdb2                8:18   1    32M  0 part  
sr0                  11:0    1 700,4M  0 rom   /media/paul/Mon disque
zram0               252:0    0   7,7G  0 disk  [SWAP]
```
```
root@vanilla:~# mount
sysfs on /sys type sysfs (rw,nosuid,nodev,noexec,relatime)
proc on /proc type proc (rw,nosuid,nodev,noexec,relatime)
udev on /dev type devtmpfs (rw,nosuid,relatime,size=8062964k,nr_inodes=2015741,mode=755,inode64)
devpts on /dev/pts type devpts (rw,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=000)
tmpfs on /run type tmpfs (rw,nosuid,nodev,noexec,relatime,size=1625064k,mode=755,inode64)
/dev/sda2 on / type btrfs (rw,relatime,space_cache=v2,subvolid=5,subvol=/)
securityfs on /sys/kernel/security type securityfs (rw,nosuid,nodev,noexec,relatime)
tmpfs on /dev/shm type tmpfs (rw,nosuid,nodev,inode64)
tmpfs on /run/lock type tmpfs (rw,nosuid,nodev,noexec,relatime,size=5120k,inode64)
cgroup2 on /sys/fs/cgroup type cgroup2 (rw,nosuid,nodev,noexec,relatime,nsdelegate,memory_recursiveprot)
pstore on /sys/fs/pstore type pstore (rw,nosuid,nodev,noexec,relatime)
efivarfs on /sys/firmware/efi/efivars type efivarfs (rw,nosuid,nodev,noexec,relatime)
bpf on /sys/fs/bpf type bpf (rw,nosuid,nodev,noexec,relatime,mode=700)
systemd-1 on /proc/sys/fs/binfmt_misc type autofs (rw,relatime,fd=29,pgrp=1,timeout=0,minproto=5,maxproto=5,direct,pipe_ino=22585)
debugfs on /sys/kernel/debug type debugfs (rw,nosuid,nodev,noexec,relatime)
mqueue on /dev/mqueue type mqueue (rw,nosuid,nodev,noexec,relatime)
hugetlbfs on /dev/hugepages type hugetlbfs (rw,relatime,pagesize=2M)
tracefs on /sys/kernel/tracing type tracefs (rw,nosuid,nodev,noexec,relatime)
fusectl on /sys/fs/fuse/connections type fusectl (rw,nosuid,nodev,noexec,relatime)
configfs on /sys/kernel/config type configfs (rw,nosuid,nodev,noexec,relatime)
/dev/sda2 on /.system/usr type btrfs (ro,relatime,space_cache=v2,subvolid=5,subvol=/)
ramfs on /run/credentials/systemd-sysusers.service type ramfs (ro,nosuid,nodev,noexec,relatime,mode=700)
/dev/sda7 on /boot type ext4 (rw,noatime,errors=remount-ro)
/dev/sda3 on /boot/efi type vfat (rw,relatime,fmask=0077,dmask=0077,codepage=437,iocharset=iso8859-1,shortname=mixed,errors=remount-ro)
/dev/sda9 on /home type btrfs (rw,relatime,space_cache=v2,subvolid=5,subvol=/)
sunrpc on /run/rpc_pipefs type rpc_pipefs (rw,relatime)
tmpfs on /run/user/1000 type tmpfs (rw,nosuid,nodev,relatime,size=1625060k,nr_inodes=406265,mode=700,uid=1000,gid=1000,inode64)
gvfsd-fuse on /run/user/1000/gvfs type fuse.gvfsd-fuse (rw,nosuid,nodev,relatime,user_id=1000,group_id=1000)
/dev/sdb1 on /media/paul/Ventoy type exfat (rw,nosuid,nodev,relatime,uid=1000,gid=1000,fmask=0022,dmask=0022,iocharset=utf8,errors=remount-ro,uhelper=udisks2)
/dev/sr0 on /media/paul/Mon disque type iso9660 (ro,nosuid,nodev,relatime,norock,check=r,map=n,blocksize=2048,uid=1000,gid=1000,dmode=500,fmode=400,iocharset=utf8,uhelper=udisks2)
portal on /run/user/1000/doc type fuse.portal (rw,nosuid,nodev,relatime,user_id=1000,group_id=1000)
binfmt_misc on /proc/sys/fs/binfmt_misc type binfmt_misc (rw,nosuid,nodev,noexec,relatime)
/var/lib/snapd/snaps/snapd_19361.snap on /snap/snapd/19361 type squashfs (ro,nodev,relatime,errors=continue,x-gdu.hide,x-gvfs-hide)
/var/lib/snapd/snaps/core20_1891.snap on /snap/core20/1891 type squashfs (ro,nodev,relatime,errors=continue,x-gdu.hide,x-gvfs-hide)
/var/lib/snapd/snaps/bare_5.snap on /snap/bare/5 type squashfs (ro,nodev,relatime,errors=continue,x-gdu.hide,x-gvfs-hide)
/var/lib/snapd/snaps/gtk-common-themes_1535.snap on /snap/gtk-common-themes/1535 type squashfs (ro,nodev,relatime,errors=continue,x-gdu.hide,x-gvfs-hide)
/var/lib/snapd/snaps/firefox_2710.snap on /snap/firefox/2710 type squashfs (ro,nodev,relatime,errors=continue,x-gdu.hide,x-gvfs-hide)
/var/lib/snapd/snaps/gnome-3-38-2004_140.snap on /snap/gnome-3-38-2004/140 type squashfs (ro,nodev,relatime,errors=continue,x-gdu.hide,x-gvfs-hide)
tmpfs on /run/snapd/ns type tmpfs (rw,nosuid,nodev,noexec,relatime,size=1625064k,mode=755,inode64)
nsfs on /run/snapd/ns/firefox.mnt type nsfs (rw)
/dev/sda2 on /var/snap/firefox/common/host-hunspell type btrfs (ro,noexec,noatime,space_cache=v2,subvolid=5,subvol=/)
root@vanilla:~# 
```
Will try to reboot anyway, probably after "sudo update grub".

**Comment by taukakao** [comment]
I'm not sure what this issue is about. 
Is this still a problem? 
If so, could you rephrase?
